app/controllers/reports_controller.rb

```diff
@@ -1,12 +1,33 @@
 class ReportsController < ApplicationController
   def call_volumes
-    @call_volumes = CallVolumes.new(form_params)
+    @call_volumes = CallVolumes.new(date_params(:call_volumes))
 
     respond_to do |format|
       format.html
       format.csv do
-        filename = "call_volume_#{@call_volumes.period}"
-        render csv: DailyCallVolumeCsv.new(@call_volumes.results), filename: filename
+        render csv: DailyCallVolumeCsv.new(@call_volumes.results), filename: 'call_volume.csv'
+      end
```

Comment:
@benbarnett @benlovell Added a commit to standise CSV naming to include ".csv" and be the name of the report - after taking with John.